deploy/provider-gcp-plugin.yaml.tmpl

```diff
@@ -28,6 +65,7 @@ spec:
       labels:
         app: csi-secrets-store-provider-gcp
     spec:
+      serviceAccountName: secrets-store-csi-driver-provider-gcp
       containers:
         - name: provider-gcp-installer
```

Comment:
Should we rename this now that it's a service and not just installing a binary?

Reply:
Good Point!